mediaEvents.js

```diff
@@ -179,6 +179,11 @@ var MediaEvents = (function () {
                 type: 'segment',
                 premature: true
             });
+
+            // Reset state at end of video
+            this.lastKnownPlaybackTime = 0;
```

Comment:
'lastKnownPlaybackTime' is set to a value just a few lines above and seemingly not used between?